deploy/kubernetes/examples/demo-pvc-file-system.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   name: demo-pvc-file-system
 spec:
-  volumeMode: Filesystem
+  volumeMode: Filesystem   # Optional. Values Filesystem\Block. The default is Filesystem.
```

Comment:
please remove `Values Filesystem\Block.`:
`# Optional. The default is Filesystem.`

Reply:
LGTM as well

Reply:
done